### DISCUSSION TITLE
Artifact storage quota has been hit - how to overcome this

### DISCUSSION THREAD

**Discussion by leshikus** [intel-ai/hdk#36] [Q&A] [answered]
For the private repositories the storage quota is imposed; I wonder what is correct way to overcome this. @alexbaden @Garra1980 

1) One way is to skip artifact creation step for the sake of the tests
2) Another way is to make the repository public
3) Also it's possible to move CI checks to the separate (public) repo

**Comment by Garra1980** [comment]
What kind of artifacts are created?

**Reply by leshikus** [reply]
1) conda cache; this is optional;
2) log files; these are optional
2) executables; to enable different ("green") checks for a build step and a tests step I need to put them into different containers; the recommend way to exchange files between containers is by using artifacts;

**Comment by leshikus** [comment]
Meanwhile I will do a simpler solution with no artifacts

**Comment by leshikus** [comment, marked as answer]
This is no longer a problem
